Packs/McAfee_Advanced_Threat_Defense/Integrations/McAfee_Advanced_Threat_Defense/McAfee_Advanced_Threat_Defense.py

```diff
@@ -41,7 +41,7 @@ def load_server_url():
 LOGIN_HEADERS = {
     'Accept': 'application/vnd.ve.v1.0+json',
     'Content-Type': 'application/json',
-    'VE-SDK-API': base64.b64encode(USERNAME + ':' + PASSWORD)
+    'VE-SDK-API': base64.b64encode(str(USERNAME + ':' + PASSWORD).encode('utf-8'))
```

Comment:
didn't see a test playbook in the build, do we have an instance to check this still works as expected?

Reply:
Yes, We have credentials and I checked it locally, with test playbook.